Routes/userRoutes.js

```diff
@@ -28,6 +29,19 @@ router.get('/getAllUsers', async (req, res) => {
     };
 })
 
+router.get('/getId', async (req, res) =>{
```

Comment:
looks like the real point of this is to check if a person is logged in, so the name should be something like "check-authentication"